src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { PersonRow } from './PeopleRows';
+
+type Props = {
+  people: People[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => (
+  <table className="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Sex</th>
+        <th>Born</th>
+        <th>Died</th>
+        <th>Mother</th>
+        <th>Father</th>
+      </tr>
+    </thead>
+    <tbody>
+      {people.map(person => (
+        <tr key={person.slug}>
+          <PersonRow person={person} />
+        </tr>
```

Comment:
I think it's ideal to keep this row inside `PersonRow` component and please rename `PeopleRow `to `PersonRow`. 

You can have your key passed to your component like this `<PersonRow person={person} key={person.slug}/>`